src/components/Drawer/Drawer.module.css

```diff
@@ -11,34 +11,29 @@
  */
 .drawer {
   display: flex;
+  flex-direction: column;
+  align-items: flex-end;
   justify-content: flex-end;
   visibility: hidden;
-  width: 100%;
+  width: 28rem;
```

Comment:
Moves sizing of the drawer to the drawer container. The housed drawer window will be a relative value